app-i18n/fcitx-chinese-addons/fcitx-chinese-addons-5.1.2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="fcitx5-chinese-addons"
+
+inherit cmake xdg
+
+DESCRIPTION="Addons related to Chinese, including IME previous bundled inside fcitx4."
+HOMEPAGE="https://github.com/fcitx/fcitx5-chinese-addons"
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_PN}-${PV}_dict.tar.xz"
+
+LICENSE="GPL-2+ LGPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="5"
+IUSE="+gui webengine +cloudpinyin +qt5 lua +opencc test"
```

Comment:
gui & qt5 will be enabled by default.. just a question, does it work fine with those USEs disabled? and no qt6 support currently?

Reply:
The configuration of fcitx-chinese-addons is on fcitx5-configtool or fcitx5-config-qt. If both uses are turned off, it will work normally.

A more complete description is: When running fcitx5-configtool or fcitx5-config-qt, the configuration interface of Addons -> Pinyin will not change. If we start it through the command line, we can see the specific prompt information.

The support for qt6 is in the master branch and has not been released yet.